.github/workflows/main.yaml

```diff
@@ -19,7 +19,6 @@ on:
 
 jobs:
   docker-image-build:
-    runs-on: ubuntu-latest
```

Comment:
Why did you remove that?

Reply:
GitHub Actions don't allow you to combine `runs-on` and `uses` in a job, because the workflow passed in `uses` decides the container(s) it is run on.